src/components/Smaug/smaug.component.js

```diff
@@ -0,0 +1,32 @@
+import {getClient} from './smaug.mock';
+import {log} from '../../utils/logging.util';
+
+
+/**
+ * Validate token.
+ *
+ * If a token does not validate. Redirect user to the provided returnUrl
+ * @todo Should the application throw a 403 instead of calling the returnURL?
```

Comment:
Jeg er for 403. retur-url er kun efter ok validering - ellers kan vi bruges til at "spamme" andre.